### PR TITLE
User correct terminology in readme

### PR DESCRIPTION
**Pull request by bdemers**
Corrected example code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,53 +82,25 @@ compile 'com.okta.spring:okta-spring-boot-starter:${okta.springboot.version}'
...
-| okta.oauth2.clientId   | N/A | The Client Id of your Okta OIDC application |
-| okta.oauth2.clientSecret   | N/A | The Client Secret of your Okta OIDC application |
-
-There are many more properties that you can optionally configure as well. Here are some examples:
-
-| Property | Default | Details |
-|----------|---------|---------|
-| okta.oauth2.audience   | api://default | The audience of your [Authorization Server](/docs/how-to/set-up-auth-server.html) |
-| okta.oauth2.groupsClaim | groups | The claim key in the Access Token's JWT that corresponds to an array of the users groups. |
-| okta.oauth2.postLogoutRedirectUri | N/A | Set to a relative or absolute URI to enable [RP-Initiated (SSO) logout](https://developer.okta.com/blog/2020/03/27/spring-oidc-logout-options). |
```
Comment (bdemers) [inline README.md:107]: Moved this section below, post logout redirect config isn't needed in the resource server section

File: README.md [hunk 2]
```diff
@@ -82,53 +82,25 @@ compile 'com.okta.spring:okta-spring-boot-starter:${okta.springboot.version}'
 
 where ${okta.springboot.version} is the latest published version in [Maven Central](https://search.maven.org/search?q=g:com.okta.spring%20a:okta-spring-boot-starter).
 
-## Supporting client side applications - OAuth Implicit flow
+## Building API Applications - Resource Server
```
Comment (bdemers) [inline README.md:85]: We probably want to restructure this Readme a little more after this gets merged in.

The Authorization Code flow is the happy path, that _should_ be the first example, then a Resource Server example, and then if at all the custom hosted example notes should just be a quick link to the examples repo.

File: README.md [hunk 3]
```diff
@@ -82,53 +82,25 @@ compile 'com.okta.spring:okta-spring-boot-starter:${okta.springboot.version}'
...
 
 You can configure your applications properties with environment variables, system properties, or configuration files. Take a look at the [Spring Boot documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html) for more details.
 
 Only these three properties are required for a web app:
 
-| Property | Default | Details |
-|----------|---------|---------|
-| okta.oauth2.issuer     | N/A | [Authorization Server](https://developer.okta.com/docs/how-to/set-up-auth-server.html) issuer URL, i.e.: https://{yourOktaDomain}/oauth2/default |
-| okta.oauth2.clientId   | N/A | The Client Id of your Okta OIDC application |
-| okta.oauth2.clientSecret   | N/A | The Client Secret of your Okta OIDC application |
```
Comment (bdemers) [inline README.md:99]: merged with table below, and added new column (so it wasn't actually removed)

File: README.md [hunk 4]
```diff
@@ -144,8 +116,8 @@ public class DemoApplication {
 	}
 
 	@GetMapping("/hello-oauth")
-	public String hello(@AuthenticationPrincipal OidcUser user) {
```
Comment (bdemers) [inline README.md:147]: This only works for OIDC flows, not resource servers

File: README.md [hunk 5]
```diff
@@ -326,6 +298,38 @@ You can configure your applications properties with environment variables, syste
 | okta.oauth2.issuer     | true | [Authorization Server](/docs/how-to/set-up-auth-server.html) issuer URL, i.e.: https://{yourOktaDomain}/oauth2/default |
 | okta.oauth2.clientId   | true | The Client Id of your Okta OIDC application |
 | okta.oauth2.clientSecret   | true | The Client Secret of your Okta OIDC application |
+| okta.oauth2.postLogoutRedirectUri | false | Set to a relative or absolute URI to enable [RP-Initiated (SSO) logout](https://developer.okta.com/blog/2020/03/27/spring-oidc-logout-options). |
+
+**NOTE**: On setting **postLogoutRedirectUri**, you will be redirected to it after the end of your session. Therefore, this resource must be available anonymously, so be sure to add it to your `HttpSecurity` configuration.
+
+<details>
```
Comment (bdemers) [inline README.md:305]: Moved the section above about the `postLogoutRedirectUri` prop into a collapsed section, as reader doesn't have enough context at this point (the full example is below).

**Review by arvindkrishnakumar-okta — APPROVED** [review]

**Pull request merged**